packages/lab/src/metric/MetricContent.css

```diff
@@ -13,17 +13,17 @@
 }
 
 .saltMetric-direction-up .saltMetricContent-indicator {
-  fill: var(--salt-status-positive-foreground);
+  fill: var(--salt-sentiment-positive-foreground-informative);
```

Comment:
@pseys is this the first time we're using informative for an icon?

Reply:
Decorative tokens should be used on icons and graphical elements only. Informative is for text only.